### DISCUSSION TITLE
Open Cover for C# .NET runs but produces no results

### DISCUSSION THREAD

**Discussion by nitinyalamoori** [OpenCover/opencover#1025] [Q&A]
Open Cover for C# .NET runs but produces no results if I get the source code and compile it. Whereas if I use the installer, I am getting coverage results. Has anyone faced this issue? If yes, please let me know how to resolve this.

**Comment by sawilde** [comment]
Which version are you using?

**Reply by nitinyalamoori** [reply]
Hi Shaun Wilde,

I reviewed the version numbers of the code and installer and found that they are different. This is the reason why I was not getting coverage data. I downloaded the latest release code and installer. Both the results are in Sync. Thanks for this.

I am having another issue. I am not sure if I have to start a new Q/A discussion.

Q: I am using the below command to generate the coverage data for my application SOMtest.exe 

OpenCover.Console.exe "-target:X:\Container\Bin\Assemblies\Debug\somtest.exe" "-searchdirs:X:\Container\Bin\Assemblies\Debug\" -register:user

There are around 652 PDB's at the search directory location, but the coverage is generated for only 7 modules. Please let me know if I am doing anything wrong in the command line.

Thanks and Regards,
Nitin

**Reply by sawilde** [reply]
It depends on what assemblies are loaded at runtime during execution - if an assembly isn't loaded then it will not attempt to gather coverage information for that assembly (then the include/exclude rules kickin)

**Reply by nitinyalamoori** [reply]
Currently, coverage is written for assemblies that are loaded during open of the application. 

This application does some processing. Upon some user clicks, it loads some other assemblies, coverage is not written for these assemblies.

**Reply by sawilde** [reply]
Are these other assemblies showing in the XML file, if so is there a reason provided?
